src/components/Node.tsx

```diff
@@ -55,15 +52,12 @@ export class UnconnectedNode extends React.Component<IExtraProps & INode & INode
             return {
                 nodeID: dbid,
 
-                parent: -1,
+                parent: 1,
```

Comment:
No, this one is different. This is an uninitialized node, and this -1 is not the -1 for missing parent, but is just there to mean uninitialized.